### DISCUSSION TITLE
MAUI: Navigating between tabbed pages

### DISCUSSION THREAD

**Discussion by ykoorb** [PrismLibrary/Prism#3125] [.NET MAUI] [answered]
Hi there,

I'm just migrating a XF app over to Maui and i'm struggling to find the equivalent of :
`await NavigationService.SelectTabAsync("TabTwo");`

In my situation, i initialize my app using something like this - 
`/MainPage?createTab=TabOne&createTab=TabTwo`

I can see there is a navigation parameter for setting the selected tab - but how do I utilise it? I've also had a tinker with the NavigationBuilder and attempted something like this :
```
NavigationService.CreateBuilder()
   .AddTabbedSegment(b => b.SelectedTab("TabTwo"))
   .Navigate();
```

But it looks to entirely reset my navigation tree

**Comment by dansiegel** [comment, marked as answer]
I believe this API was added after the last public preview and is currently available for Commercial Plus license holders.

**Reply by ykoorb** [reply]
Hey @dansiegel - thanks for your quick response

I currently fall into the Community bracket - is it likely this feature will ever become part of that offering?

**Reply by brianlagunas** [reply]
Yes, you just have to wait longer to get it.

**Reply by ykoorb** [reply]
OK, that's very reasonable :)

In the meantime I shall have to find a workaround then! 

Thanks for your assistance and also thanks for the framework in general, it's certainly saved me a lot of time and effort

**Reply by RobFrancisAu** [reply]
EDIT : It's in 9.0.401-pre. Thanks.

@brianlagunas, can you give an Eta on this please?
I've also just ran into this issue migrating an old Xamarin codebase over to Maui. Appreciate the effort from you guys !
